service/awsconfig/v10/ebs/error.go

```diff
@@ -0,0 +1,27 @@
+package ebs
+
+import (
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/giantswarm/microerror"
+)
+
+var invalidConfigError = microerror.New("invalid config")
+
+// IsInvalidConfig asserts invalidConfigError.
+func IsInvalidConfig(err error) bool {
+	return microerror.Cause(err) == invalidConfigError
+}
+
+// IsVolumeNotFound asserts volume not found error from upstream's API code.
+func IsVolumeNotFound(err error) bool {
+	aerr, ok := err.(awserr.Error)
+	if !ok {
+		return false
+	}
+	// TODO Find constant in the Go SDK for the error code.
+	if aerr.Code() == "NotFound" {
+		return true
+	}
```

Comment:
Again the error code foo is a pain. This still needs some tidying up.

Reply:
I know. I would like to have a private error though just to have an error and its matcher.

Reply:
Done. I think its better now.